website/pages/docs/k8s/crds.mdx

```diff
@@ -0,0 +1,482 @@
+---
+layout: docs
+page_title: Consul Custom Resource Definitions (Beta)
+sidebar_title: Custom Resource Definitions <sup>(Beta)</sup>
+description: >-
+  Consul now supports managing configuration entries via Kubernetes Custom Resources.
+  These custom resource can be used to manage the configuration for workloads
+  deployed within the cluster.
+---
+
+# Custom Resource Definitions <sup>(beta)</sup>
+
+## Overview
+
+-> **This feature is currently in Beta.** It requires consul-helm >= 0.25.0 and
+consul >= 1.8.4.
```

Comment:
Should we add the required conusl-k8s version or are we piggybacking on the defaults in helm?

Reply:
We should probably decide on this everywhere. In the values.yaml I do pin consul-k8s but in our compatibility guides we just use consul-helm and essentially the consul-k8s version always needs to be the version from that helm release.